client/src/main/java/com/paypal/selion/internal/reports/runtimereport/JsonRuntimeReporterHelper.java

```diff
@@ -275,10 +275,8 @@ public synchronized void writeJSON(String outputDirectory, boolean bForceWrite)
         logger.entering(new Object[] { outputDirectory, bForceWrite });
 
         long currentTime = System.currentTimeMillis();
-        if (!bForceWrite) {
-            if (currentTime - previousTime < ONE_MINUTE) {
-                return;
-            }
+        if (!bForceWrite && currentTime - previousTime < ONE_MINUTE) {
```

Comment:
Does this needs parenthesis to control order of operations? It sure is hard to read.